site/docs/.vuepress/configs/en.ts

```diff
@@ -74,10 +74,6 @@ export const localeEn: LocaleConfig<DefaultThemeLocaleData> = {
                 text: "Error Handling",
                 link: "/guide/errors.html",
               },
-              {
-                text: "Inline Queries",
-                link: "/guide/inline-queries.html",
-              },
```

Comment:
Can we have a redirect here?

Reply:
I don’t know.

Reply:
Most likely this can be done in the netlify config. I do not see a way to perform redirects from VuePress. It consists of static files only, so redirects would have to be performed JS-side, which is far from optimal.